docs/build/tutorials/migrating-ethereum-app-to-kaia.mdx

```diff
@@ -0,0 +1,197 @@
+
+          ```js
+            const { JsonRpcProvider } = require("@kaiachain/ethers-ext/v6");
+
+            // const SEPOLIA_TESTNET_RPC_URL = 'https://ethereum-sepolia-rpc.publicnode.com'
+            const KAIROS_TESTNET_RPC_URL = 'https://public-en-kairos.node.kaia.io/'
+            const provider = new JsonRpcProvider(KAIROS_TESTNET_RPC_URL);
+
+            async function getKaiaBlockNumber() {
+              // Get the current block number
+              const blockNumber = await provider.getBlockNumber();
+              console.log("Current Kaia block number:", blockNumber);
+
+            }
+
+          getKaiaBlockNumber()
+
+          ```
+
+          **Contract Data**
+
+        ```js 
+          const ethers = require("ethers");
+
+          const provider = new ethers.providers.JsonRpcProvider("https://public-en-kairos.node.kaia.io");
+
+          /* compiled in remix.ethereum.org (compiler: 0.8.18, optimizer: false)
+          // SPDX-License-Identifier: UNLICENSED
+          pragma solidity ^0.8.13;
+
+          contract Counter {
+              uint256 public number;
+              event SetNumber(uint256 number);
+
+              constructor(uint256 initNumber) {
+                  number = initNumber;
+              }
+
+              function setNumber(uint256 newNumber) public {
+                  number = newNumber;
+                  emit SetNumber(number);
+              }
+
+              function increment() public {
+                  number++;
+                  emit SetNumber(number);
+              }
+          }
+          */
+          const abi = '[{"inputs":[{"internalType":"uint256","name":"initNumber","type":"uint256"}],"stateMutability":"nonpayable","type":"constructor"},{"anonymous":false,"inputs":[{"indexed":false,"internalType":"uint256","name":"number","type":"uint256"}],"name":"SetNumber","type":"event"},{"inputs":[],"name":"increment","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[],"name":"number","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"uint256","name":"newNumber","type":"uint256"}],"name":"setNumber","outputs":[],"stateMutability":"nonpayable","type":"function"}]';
+          const contractAddr = "0x95Be48607498109030592C08aDC9577c7C2dD505";
+
+          async function main() {
+            const counter = new ethers.Contract(contractAddr, abi, provider);
+            const number = await counter.number();
+            console.log("number", number.toString());
+          }
+
+          main();
+        ```
+
+        **B. Write to the blockchain**
+
+      ```js
+      const ethers = require("ethers");
+      const { Wallet } = require("@kaiachain/ethers-ext/v6");
+
+      const senderAddr = "0x24e8efd18d65bcb6b3ba15a4698c0b0d69d13ff7";
+      const senderPriv = "0x4a72b3d09c3d5e28e8652e0111f9c4ce252e8299aad95bb219a38eb0a3f4da49";
```

Comment:
Hi @ayo-klaytn , i would suggest to better avoid providing Private keys even in the documentations even though they are dummy ones.

So we can either give them as place holders or as environment variables. We can mention users to replace them .